pkg/planner/core/indexmerge_path.go

```diff
@@ -761,6 +762,14 @@ func (ds *DataSource) generateMVIndexMergePartialPaths4And(normalPathCnt int, in
 	// step2: mapping index merge conditions into possible mv index path
 	mvAndPartialPath := make([]*util.AccessPath, 0, len(possibleMVIndexPaths))
 	usedAccessCondsMap := make(map[string]expression.Expression, len(indexMergeConds))
+	// fill the possible indexMergeConds down to possible index merge paths. If two index merge path
+	// share same accessFilters, pick the one with minimum countAfterAccess.
+	type record struct {
+		originOffset     int
+		paths            []*util.AccessPath
+		countAfterAccess float64
+	}
+	mm := make(map[string]*record, 0)
```

Comment:
Let's use a more meaningful variable name or add some comments about what this map is for.

Reply:
And the `, 0` is actually redundant.